### PR TITLE
AWS lambda refactor + travis CICD

### PR DESCRIPTION
**Pull request by nikitabelonogov**

### CODE REVIEW

**Comment by wlad031** [comment]
I guess that we can use only one check - on PRs. And disable pushing to main branch.

File: lambda.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (wlad031) [inline lambda.py:1]: Can we still use run.py and import it here? lambda.py will be just an entrypoint.

**Comment by nikitabelonogov** [comment]
@wlad031 master branch is protected now.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+language: python
```
Comment (wlad031) [inline .travis.yml:1]: Do we use python 2.7?

Comment (nikitabelonogov) [reply .travis.yml:1]: yes, is it a problem?
we can use 3.6

File: lambda.py (hunk 1 above)
Comment (nikitabelonogov) [reply lambda.py:1]: Yes.

File: .travis.yml (hunk 2 above)
Comment (wlad031) [reply .travis.yml:1]: It's a problem for such hipsters 💩

**Review by wlad031 — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (nikitabelonogov) [reply .travis.yml:1]: True.

**Pull request merged**